org.eclipse.scout.rt.dataobject.test/src/test/java/org/eclipse/scout/rt/dataobject/id/IdInventoryTest.java

```diff
@@ -35,14 +36,14 @@ public void testGetIdClass() {
   @Test
   public void testGetTypeNameByClass() {
     assertEquals("scout.FixtureUuId", m_inventory.getTypeName(FixtureUuId.class));
-    assertNull(m_inventory.getTypeName(FixtureLongId.class));
+    assertNull(m_inventory.getTypeName(FixtureLongIdWithoutTypeName.class));
```

Comment:
you could add another testcase for FixtureLongId

Reply:
done